packages/core/admin/ee/admin/pages/SettingsPage/pages/SingleSignOn/tests/__snapshots__/index.test.js.snap

```diff
@@ -76,6 +76,35 @@ exports[`Admin | ee | SettingsPage | SSO renders and matches the snapshot 1`] =
   flex: 1;
 }
 
+.c47 {
```

Comment:
I still think we don't need these snapshot tests - what value do you think they add?

Reply:
I removed the snapshot test